### ISSUE TITLE
Missing alt text from github icons on `openlifesci.org/people`

### ISSUE THREAD

**Issue by da5nsy** [open-life-science/open-life-science.github.io#634]
What it says in the title really!
If I get a chance to dig into this and try to fix it I will...

![image](https://github.com/open-life-science/open-life-science.github.io/assets/3739866/69e38561-439e-4725-a299-ab6fbf1207ed)

**Comment by da5nsy** [comment]
Oof this is not a well written github issue, sorry!
I have no idea how I encountered this 😢 

My guess is that `{{ user.name }}` ([here](https://github.com/open-life-science/open-life-science.github.io/blob/main/_includes/people.html#L9)) needs to be switched to `{{ username }}` (as [here](https://github.com/open-life-science/open-life-science.github.io/blob/main/_includes/people.html#L1C35-L1C49)) or `{{ user.first-name }} {{ user.last-name }}` (as [here](https://github.com/open-life-science/open-life-science.github.io/blob/main/_includes/people.html#L14C31-L14C73)).
